packages/functions/src/migrations.ts

```diff
@@ -211,3 +212,39 @@ export const clearDeletedCustomersRegistrationAndCategories = functions
 
     await batch.commit();
   });
+
+export const normalizeExistingEmails = functions
+  .region(__functionsZone__)
+  .https.onCall(
+    async ({ organization }: { organization: string }, { auth }) => {
+      if (!(await checkUser(organization, auth))) throwUnauth();
+
+      const db = admin.firestore();
+
+      const batch = db.batch();
+
+      const customres = await db
+        .collection(Collection.Organizations)
+        .doc(organization)
+        .collection(OrgSubCollection.Customers)
+        .get();
+
+      customres.forEach((customer) => {
+        const data = customer.data() as CustomerFull;
+        if (!data.email) return;
```

Comment:
Let's try to avoid any `set` invocation unless it's really strictly needed.

I propose this change:

```
        const { email } = data;
        const emailNormalized = normalizeEmail(data || '');
        if (email !== emailNormalized) {
          batch.set(
            customer.ref,
            { email: emailNormalized },
            { merge: true }
          );
        }
```